lib/commands/test-init.ts

```diff
@@ -37,6 +37,25 @@ class TestInitCommand implements ICommand {
 				'save-dev': true,
 				optional: false,
 			});
+
+			let modulePath = path.join(projectDir, "node_modules", mod);
+			let modulePackageJsonPath = path.join(modulePath, "package.json");
+			let modulePackageJsonContent = this.$fs.readJson(modulePackageJsonPath);
+			let modulePeerDependencies = modulePackageJsonContent.peerDependencies || {};
+
+			for (let peerDependency in modulePeerDependencies) {
+				let dependencyVersion = modulePeerDependencies[peerDependency] || "*";
+
+				// catch errors when a peerDependency is already installed
+				// e.g karma is installed; karma-jasmine depends on karma and will try to install it again
+				try {
+					await this.$npm.install(`${peerDependency}@${dependencyVersion}`, projectDir, {
+						'save-dev': true
```

Comment:
You can disregard this. I didn't see the reason to duplicate them from `peerDependencies` - they're already present and installed from their presence there. Why add more changes to the package.json